roles/storage-glusterfs/defaults/main.yml

```diff
@@ -3,3 +3,4 @@ action: "provision"
 heketi_route_name: "heketi-storage"
 heketi_namespace: "app-storage"
 heketi_secret_name: "heketi-storage-admin-secret"
+external_provisioner: false
```

Comment:
I opted for false because the glusterfile provisioner is still really really new and relatively untested. I'd be more comfortable setting it as the default when it's had more time to mature.

Reply:
ok